File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/domain/Models/SeatType.cs
using System;

namespace ConferenceManagement
{
    public class SeatType
    {
        public Guid Id { get; private set; }
        public SeatTypeInfo Info { get; set; }
        public int Quantity { get; set; }

        public SeatType(Guid id, SeatTypeInfo info)
        {
            Id = id;
            Info = info;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/model/SeatType.java
package com.microsoft.conference.management.domain.model;

public class SeatType {
    private String id;
    private SeatTypeInfo seatTypeInfo;
    private int quantity;

    public SeatType() {
    }

    public SeatType(String id, SeatTypeInfo seatTypeInfo) {
        this.id = id;
        this.seatTypeInfo = seatTypeInfo;
    }

    public String getId() {
        return this.id;
    }

    public SeatTypeInfo getSeatTypeInfo() {
        return this.seatTypeInfo;
    }

    public int getQuantity() {
        return this.quantity;
    }

    public void setId(String id) {
        this.id = id;
    }

    public void setSeatTypeInfo(SeatTypeInfo seatTypeInfo) {
        this.seatTypeInfo = seatTypeInfo;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}
